docs/api-reference/server/xviz-server-middleware-stack.md

```diff
@@ -0,0 +1,27 @@
+# XVIZServerMiddlewareStack
+
+XVIZServerMiddlewareStack conforms to the middleware interface and routes messages through the
```

Comment:
Maybe link `Middleware Interface` in `overview-middleware.md`?  you have so many other things nicely linked and it doesn't look like this is linked from anywhere else.  I would probably mention it from the main `overview.md` here.

Reply:
will add both here to overview & stack on the main page.  The nuance her, which i will explain is the middleware interface is really shared between them all.  This piece just manages the flow across the registered components.  But it's worth making that clearer.